packages/react-app-revamp/lib/analytics/participants/index.ts

```diff
@@ -25,3 +22,25 @@ const saveToAnalyticsContestParticipantsV3 = async (options: SaveToAnalyticsCont
     }
   }
 };
+
+export const addUserActionForAnalytics = async (options: SaveToAnalyticsContestParticipantsOptions) => {
+  await saveToAnalyticsContestParticipantsV3(options);
+};
+
+export const saveUpdatedProposalsStatusToAnalyticsV3 = async (proposal_ids: string[], deleted: boolean) => {
+  if (isSupabaseConfigured) {
+    const config = await import("@config/supabase");
+    const supabase = config.supabase;
+
+    for (let proposal_id of proposal_ids) {
+      const { error: updateError } = await supabase
+        .from("analytics_contest_participants_v3")
+        .update({ deleted: deleted })
+        .eq("proposal_id", proposal_id);
```

Comment:
We should also query for `contest_address` and `network_name` here just in case there are duplicate proposals on different contests.